.github/workflows/test.yml

```
@@ -32,3 +32,44 @@ jobs:
     - run: python -m pip install .
     - run: python -m pip install pytest
     - run: python -m pytest
+
+  test-aarch64:
+    name: "Test ${{ matrix.pyver }} aarch64"
+    strategy:
+      matrix:
+        pyver: [cp39-cp39]
+        arch: [aarch64]
+      fail-fast: false
+    runs-on: ubuntu-latest
+    env:
+      py: /opt/python/${{ matrix.pyver }}/bin/python
+      img: quay.io/pypa/manylinux2014_${{ matrix.arch }}
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+    - name: Set up QEMU
+      id: qemu
+      uses: docker/setup-qemu-action@v1
+    - name: Install tools
+      run: |
+        docker run --rm -v ${{ github.workspace }}:/ws:rw --workdir=/ws \
+          ${{ env.img }} ${{ env.py }} -m pip install -U  pip build
+    - name: Install dependencies
+      run: |
+            docker run --rm -v ${{ github.workspace }}:/ws:rw --workdir=/ws \
+            ${{ env.img }} \
+            bash -exc '${{ env.py }} -m venv .env && \
+            source .env/bin/activate && \
+            python -m pip install --upgrade pip setuptools wheel && \
+            deactivate'
```

Stylistic choice (so it's up to @HDembinski), but the `/`'s make this harder to read and are easier to forget when modifying this, IMO:

```suggestion
      run: >-
            docker run --rm -v ${{ github.workspace }}:/ws:rw --workdir=/ws
            ${{ env.img }}
            bash -exc '${{ env.py }} -m venv .env &&
            source .env/bin/activate &&
            python -m pip install --upgrade pip setuptools wheel &&
            deactivate'
```

---

I agree with Henry :)